### PR TITLE
Mobile Social Media Banner

### PR DESCRIPTION
**Pull request by SWKineo**
# Pull Request Process

## Describe the changes you've made to resolve the issue

- Split SocialMediaBanner component into subcomponents in the same file
- Moved CSS grid formatting into desktop subcomponent
- Added a subcomponent for mobile
- Styled mobile version
- Fixed styling of desktop version to match how it looked prior to this refactor

### Before
image
image

### After
image
image

## Code Review Checklist

- [ ] Have the package.json or package-lock.json been changed? If so, does the pull request description say why?
- [x] Has the UI for mobile and desktop been changed, did you test on IphoneXr and desktop views?
- [x] Does the UI match the requirements in the ticket and designs in figma; does it pass the visual smell test?
- [ ] Have TypeScript files been changed, does the TypeScript pass the smell test?
- [ ] If you don't feel super confident in your review, did you assign someone more senior to double check?

### CODE REVIEW

File: app/components/SocialMediaBanner.tsx [hunk 1]
```diff
@@ -3,51 +3,87 @@ import FB from "../../public/fb.svg";
 import Image from "next/image";
 import LinkedIN from "../../public/libanner.svg";
 import Ozzie from "../../public/Ozzie Crutches SVG 2.png";
+import OzzieMobile from "../../public/Ozzie Peeking In.png";
```
Comment (YolineB) [inline app/components/SocialMediaBanner.tsx:6]: We should remove the spacing for the file name

File: app/components/SocialMediaBanner.tsx [hunk 2]
```diff
@@ -3,51 +3,87 @@ import FB from "../../public/fb.svg";
...
+        <DesktopBanner />
+        <MobileBanner />
+      </div>
+    </div>
+  );
+}
+
+function DesktopBanner() {
+  return (
+    <div className="hidden md:grid grid-rows-2 md:gap-x-6 md:grid-flow-col w-[45%] px-3 py-4">
```
Comment (YolineB) [inline app/components/SocialMediaBanner.tsx:26]: Let's do `w-[65%]`, it looks a little better w/ responsive changes

File: app/components/SocialMediaBanner.tsx [hunk 3]
```diff
@@ -3,51 +3,87 @@ import FB from "../../public/fb.svg";
...
+      <div className="row-span-3 ...">
```
Comment (YolineB) [inline app/components/SocialMediaBanner.tsx:27]: Add a `self-center` and remove the `...` that was just an accidental copy from tailwind's site

File: app/components/SocialMediaBanner.tsx [hunk 4]
```diff
@@ -3,51 +3,87 @@ import FB from "../../public/fb.svg";
...
-            <a href="https://www.yelp.com/biz/luskin-orthopaedic-institute-for-children-los-angeles">
-              <Image src={Yelp} alt="Yelp Logo " />
-            </a>
-          </div>
+        <div className="text-md text-center">
+          <p>
+            Join us in our mission to transform musculoskeletal care for all,
+            one child at a time,
+          </p>{" "}
+          <p>through education, research, and unwavering support.</p>
```
Comment (YolineB) [inline app/components/SocialMediaBanner.tsx:41]: Let's have this under one paragraph tag

File: app/components/SocialMediaBanner.tsx [hunk 5]
```diff
@@ -3,51 +3,87 @@ import FB from "../../public/fb.svg";
...
-          </div>
-          <div className="text-md text-center">
-            <p>
-              Join us in our mission to transform musculoskeletal care for all,
-              one child at a time,
-            </p>{" "}
-            <p>through education, research, and unwavering support.</p>
-          </div>
+      </div>
+      <div className="hidden md:block col-span-1 ..">
```
Comment (YolineB) [inline app/components/SocialMediaBanner.tsx:32]: We can remove the two dots here as well

**Review by YolineB — CHANGES_REQUESTED** [review]

File: app/components/SocialMediaBanner.tsx [hunk 6]
```diff
@@ -2,52 +2,88 @@ import React from "react";
...
+      <SocialMediaIcons />
+    </div>
+  );
+}
+
+function SocialMediaIcons() {
+  return (
+    <div className="row-span-1 .. pt-8 pb-10 md:pt-0 md:pb-0 px-10 md:px-3">
+      <div className="flex flex-row min-h-[75%] justify-center items-center space-x-4 ..">
+        <a href="https://www.instagram.com/orthokids_losangeles/ ..">
```
Comment (YolineB) [inline app/components/SocialMediaBanner.tsx:71]: Oops! broken link for facebook, should be this: https://www.facebook.com/LuskinOIC/

**Review by YolineB — CHANGES_REQUESTED** [review]

File: app/components/SocialMediaBanner.tsx (hunk 6 above)
Comment (SWKineo) [reply app/components/SocialMediaBanner.tsx]: Fixed! I also noticed the X link (https://twitter.com/i/flow/login?redirect_after_login=%3FLuskinOIC) forces the user to log in every time, so I replaced it with this: https://twitter.com/LuskinOIC

**Review by YolineB — APPROVED** [review]
LGTM!

**Review by jtorreggiani — APPROVED** [review]

**Pull request merged**